README.md

```diff
@@ -1,143 +1,34 @@
-<div align="center"><img width="1007" alt="plasm_web3" src="https://user-images.githubusercontent.com/6259384/72399615-0e1cbb80-378a-11ea-91af-c1dbdde345f5.png"></div>
+# Astar Network &middot; [![GitHub license](https://img.shields.io/badge/license-GPLv3-blue)](#LICENSE) [![CI Status](https://github.com/PlasmNetwork/Plasm/workflows/Integration/badge.svg)](https://github.com/PlasmNetwork/Plasm/actions) 
```

Comment:
We should change the repo name as well from `https://github.com/PlasmNetwork/Plasm/actions` to `https://github.com/PlasmNetwork/Astar/actions`